### PR TITLE
bpo-45411: Update mimetypes.py

### PR DESCRIPTION
**Pull request by Josephine-Marie**
.vtt and .srt files are common subtitle files, beneficial for people who use python for web development.

https://bugs.python.org/issue45411

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2021-10-08-11-29-29.bpo-45411.4jR--U.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add Subtitle file extensions for enhancing web deleopment with Python!
```
Comment (sbmooc) [inline Misc/NEWS.d/next/Library/2021-10-08-11-29-29.bpo-45411.4jR--U.rst]: It looks like there is a typo here. Should be development

Comment (Josephine-Marie) [reply Misc/NEWS.d/next/Library/2021-10-08-11-29-29.bpo-45411.4jR--U.rst]: thanks!

File: Misc/NEWS.d/next/Library/2021-10-07-13-48-53.bpo-00000.1md5vg.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Add MIME types for Subtitle-Files.
```
Comment (JulienPalard) [inline Misc/NEWS.d/next/Library/2021-10-07-13-48-53.bpo-00000.1md5vg.rst]: This file can be removed as you did a proper NEWS entry with a bpo id :)

File: Misc/NEWS.d/next/Library/2021-10-08-11-29-29.bpo-45411.4jR--U.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Add Subtitle file extensions for enhancing web development with Python!
```
Comment (JulienPalard) [inline Misc/NEWS.d/next/Library/2021-10-08-11-29-29.bpo-45411.4jR--U.rst]: I'd prefer it more precise, tell which file extensions, tell about it's in the mimetypes module, and you can remove the part about web dev: having those extensions recognized by mimetypes is nice everywhere, not just for web dev ;)

Comment (Josephine-Marie) [reply Misc/NEWS.d/next/Library/2021-10-08-11-29-29.bpo-45411.4jR--U.rst]: Okay, hope this is better and not too short

**Review by JulienPalard — APPROVED** [review]

**Comment by JulienPalard** [comment]
Thanks @Josephine-Marie for your first PR in Python!

**Pull request merged**